### ISSUE TITLE
[Usable – Make Code – Header] Narrator announces same information twice while expanding header section controls. 

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3608]
**User Experience:**
Screen reader user will be confused, if narrator is reading same information two times and also it will take time to navigate another control over the screen.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Select code editor language, Help, More" option appearing in header section and select it.
6. Verify the issue by using screen reader.

**Actual Result:**
Narrator announces same information twice while expanding header section controls like(Select code editor language, Help, More) and it reads as 'Help menu item expanded 7 of 9.... Help menu item expanded 7 of 9'. Similarly for others.

**Expected Result:**
Narrator should should not announce same information twice while expanding header section controls like(Select code editor language, Help, More), it should read only once as 'Select code editor language menu item 5 of 9, Help menu item expanded 7 of 9, More menu item expanded 8 of 9'.

**Note:**
This issue does not exists on Firefox & Chrome.
same issue exist under download option pop up.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)
Screen Reader: Narrator

**Comment by Shalini3457** [comment]
Labels:
#A11yUsable, #Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

[Usable_Header_Twice.zip](https://github.com/microsoft/pxt-microbit/files/5499667/Usable_Header_Twice.zip)

**Comment by jwunderl** [comment]
Appears fixed in edge
